src/test/java/org/mongoflink/source/MongoBatchSourceTest.java

```diff
@@ -64,13 +66,13 @@ public void testRead() throws Exception {
         ListSink<String> sink = new ListSink<>();
         env.fromSource(mongoSource, WatermarkStrategy.noWatermarks(), "mongo_batch_source")
                 .returns(String.class)
-                .addSink(sink);
+                .print("heihei");
```

Comment:
Maybe we should add an extra test for projection instead of reusing the existing one?

Reply:
ok. i will add an extra test